### ISSUE TITLE
i cant install ibus-avro

### ISSUE THREAD

**Issue by mrX215** [sarim/ibus-avro#152]
mr@kali:~$ sudo apt install ibus-avro
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package ibus-avro
mr@kali:~$

**Comment by gunnarhj** [comment]
Which distro are you on? Are you possibly on Debian stable?

**Comment by sarim** [comment]
there is a `kali` in the prompt ..........

**Comment by gunnarhj** [comment]
Right, @sarim, good catch. In that case I suspect that `-backports` has not been enabled in `/etc/apt/sources.list`.

**Comment by mrX215** [comment]
@sarim 
assalamualaikum. what can i do for it.
please support me & suggest me.

**Comment by gunnarhj** [comment]
@mrX215: Can you show us your `sources.list` file by running:

    cat /etc/apt/sources.list

**Comment by AnupamMitra** [comment]
I have installed ibus-Avro in Elementary OS. But when I am typing in bengali nothing is coming out in Libre Office Writer file and not even in the browser. Though pressing Ctrl+Space Bar I could toggle between English and Avro (Bengali). Did I make any mistake?
